test/integration/targets/test.yaml

```diff
@@ -0,0 +1,12 @@
+- hosts: all
+  module_defaults:
+    myos_facts:
+      gather_subset: min
```

Comment:
`gather_subset` can be `['!min', '!all']` so that no other facts will be pulled except the resource which will save us some test runtime?